### PR TITLE
Append 'User' instead of 'Account'

### PR DESCRIPTION
**Pull request by shoekstra**
'User' is more inline with AWS terminology and matches how the Terraform resources are called while 'Account' refers to the actual AWS account.

### CODE REVIEW

**Review by stefanwb — DISMISSED** [review]

**Comment by shoekstra** [comment]
Please do not merge this just yet, @svanharmelen and I discussed this on Slack and we want to check the implications before merging.

**Pull request closed without merging**
